subt/main.py

```diff
@@ -154,7 +154,7 @@ def __init__(self, config, bus):
         self.walldist = config['walldist']
         self.timeout = timedelta(seconds=config['timeout'])
         self.symmetric = config['symmetric']  # is robot symmetric?
-        self.dangerous_dist = config.get('dangerous_dist', 0.4)
+        self.dangerous_dist = config.get('dangerous_dist', 0.3)
```

Comment:
why do you have config files if you change the default?? If nothing else it breaks the Virtual robots

Reply:
p.s. not speaking of Eduro ...

Reply:
so, shall we revert it?